tez-dag/src/main/java/org/apache/tez/dag/app/dag/impl/DAGSchedulerNaturalOrderControlled.java

```diff
@@ -154,8 +154,7 @@ private void processDownstreamVertices(Vertex vertex) {
     List<Vertex> newlyScheduledVertices = Lists.newLinkedList();
     Map<Vertex, Edge> outputVertexEdgeMap = vertex.getOutputVertices();
     for (Vertex destVertex : outputVertexEdgeMap.keySet()) {
-      if (vertexAlreadyScheduled(destVertex)) { // Nothing to do if already scheduled.
-      } else {
+      if (!vertexAlreadyScheduled(destVertex)) { // Nothing to do if already scheduled.
```

Comment:
you can delete the comment "Nothing to do" from here as it corresponds to the deleted branch

Reply:
done